fsharp/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fsharp:10.10-netcore AS build
```

Comment:
This image is still based on dotnet-core.

Reply:
you can use `dotnet` image from microsoft to compile `F#` ?

Reply:
Yes, I think the fsharp image is the build from fsharp.org, but F# is released as part of the dotnet-sdk, as well.